service/src/main/java/skills/auth/form/CreateAccountController.groovy

```diff
@@ -135,11 +135,10 @@ class CreateAccountController {
     @RequestMapping(value = "/grantFirstRoot", method = [RequestMethod.POST, RequestMethod.PUT])
     void grantFirstRoot(HttpServletRequest request) {
         SkillsValidator.isTrue(!userAuthService.rootExists(), 'A root user already exists! Granting additional root privileges requires a root user to grant them!')
-        SkillsValidator.isNotNull(request.getUserPrincipal(), 'Granting the first root user is only available in PKI mode, but it looks like the request was not made by an authenticated account!')
+        SkillsValidator.isNotNull(request.getUserPrincipal(), 'Granting the first root user is only available in SAML & PKI modes, but it looks like the request was not made by an authenticated account!')
         userAuthService.grantRoot(request.getUserPrincipal().name)
     }
 
-    @Conditional(SecurityMode.FormAuth)
```

Comment:
Instead of removing `@Conditional(SecurityMode.FormAuth)` annotation please change to `@Conditional(SecurityMode.FormOrSAML2Auth)`

Reply:
Done